piq/msid.py

```diff
@@ -278,6 +279,17 @@ def _msid_descriptor(x: np.ndarray, ts: np.ndarray = np.logspace(-1, 1, 256), k:
     Returns:
         normed_msidx: normalized msid descriptor
     """
+    try:
+        import scipy
+    except ImportError:
+        raise ImportError("Scipy is required for computation of the Geometry Score but not installed. "
+                          "Please install scipy using the following command: pip install --user scipy")
+
+    recommended_scipy_version = "1.3.3"
+    if _version_tuple(scipy.__version__) < _version_tuple(recommended_scipy_version):
+        warn(f'Scipy of version {scipy.__version__} is used while version >= {recommended_scipy_version} is '
+             f'recommended. Consider updating scipy to avoid potential problems.')
```

Comment:
Can we specify exactly which problems may happen when using older versions?

Reply:
It was introduced in #237. @zakajd @snk4tr, what was the problem there with older `scipy`?
I remember only slow processing in case of `1.4` version.

Reply:
Yep, this recommendation was motivated by a long compute time with older versions.

Reply:
Resolved